tests/cogs/test_new_user_join.py

```diff
@@ -0,0 +1,71 @@
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+from otter_welcome_buddy.cogs import new_user_joins
+
+
+@pytest.fixture
+def mock_bot():
```

Comment:
Can u move these fixtures to [`conftest file`](https://docs.pytest.org/en/6.2.x/fixture.html#conftest-py-sharing-fixtures-across-multiple-files) to make them reusable for other tests?